pkg/resourcegenerator/core/pod.go

```diff
@@ -58,27 +66,30 @@ func CreateApplicationContainer(application *skiperatorv1alpha1.Application, opt
 	}
 }
 
-//func CreateJobContainer(skipJob *skiperatorv1alpha1.SKIPJob) corev1.Container {
-//	return corev1.Container{
-//		Name:            util.ResourceNameWithHash(skipJob.Name, skipJob.Kind),
-//		Image:           skipJob.Spec.Container.Image,
-//		ImagePullPolicy: corev1.PullAlways,
-//		Command:         skipJob.Spec.Container.Command,
-//		SecurityContext: &corev1.SecurityContext{
-//			Privileged:               util.PointTo(false),
-//			AllowPrivilegeEscalation: util.PointTo(false),
-//			ReadOnlyRootFilesystem:   util.PointTo(true),
-//			RunAsUser:                util.PointTo(util.SkiperatorUser),
-//			RunAsGroup:               util.PointTo(util.SkiperatorUser),
-//		},
-//		EnvFrom:        getEnvFrom(skipJob.Spec.Container.EnvFrom),
-//		Resources:      getResourceRequirements(skipJob.Spec.Container.Resources),
-//		Env:            skipJob.Spec.Container.Env,
-//		ReadinessProbe: getProbe(skipJob.Spec.Container.Readiness),
-//		LivenessProbe:  getProbe(skipJob.Spec.Container.Liveness),
-//		StartupProbe:   getProbe(skipJob.Spec.Container.Startup),
-//	}
-//}
+func CreateJobContainer(skipJob *skiperatorv1alpha1.SKIPJob, volumeMounts []corev1.VolumeMount) corev1.Container {
+	return corev1.Container{
+		Name:            skipJob.HashedName(),
+		Image:           skipJob.Spec.Container.Image,
+		ImagePullPolicy: corev1.PullAlways,
+		Command:         skipJob.Spec.Container.Command,
+		SecurityContext: &corev1.SecurityContext{
+			Privileged:               util.PointTo(false),
+			AllowPrivilegeEscalation: util.PointTo(false),
+			ReadOnlyRootFilesystem:   util.PointTo(true),
+			RunAsUser:                util.PointTo(util.SkiperatorUser),
+			RunAsGroup:               util.PointTo(util.SkiperatorUser),
```

Comment:
You have `seccompProfile` on pod-level. I'm only missing `capabilities` drop all?

https://learn.snyk.io/lesson/container-does-not-drop-all-default-capabilities

Reply:
I only set these according to what rules we have with Gatekeeper. I can add the capabilities too, no reason not to, I presume!

Reply:
It's probably for the best, as it's part of [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/). I noticed we don't have it on Application either, so if you would please try to add it to Jobs and we can use that as a test case for later adding it to Application :pray:

Reply:
Created a new constant for default least privilege container security context and used that one for Jobs for now!